container.go

```diff
@@ -239,41 +218,55 @@ func terminate(profile *Profile, all bool) error {
 	} else {
 		f.Add("label", "com.viam.canon.profile="+profile.name+"/"+profile.Arch)
 	}
-	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{Filters: f})
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true, Filters: f})
 	if err != nil {
 		return err
 	}
 	if len(containers) > 1 && !all {
 		return errors.New("multiple matching containers found, please retry with '--all' option")
 	}
 	for _, c := range containers {
-		fmt.Printf("terminating %s\n", c.Labels["com.viam.canon.profile"])
-		err = multierr.Combine(err, cli.ContainerStop(context.Background(), c.ID, container.StopOptions{}))
+		if terminate {
+			fmt.Printf("terminating %s\n", c.Labels["com.viam.canon.profile"])
+			err = errors.Join(err, cli.ContainerRemove(ctx, c.ID, container.RemoveOptions{Force: true}))
+		} else {
+			fmt.Printf("stopping %s\n", c.Labels["com.viam.canon.profile"])
+			err = errors.Join(err, cli.ContainerStop(ctx, c.ID, container.StopOptions{}))
+		}
 	}
 	return err
 }
 
-func list() error {
+func list(ctx context.Context) error {
 	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
 	if err != nil {
 		return err
 	}
 	f := filters.NewArgs(filters.Arg("label", "com.viam.canon.profile"))
 
-	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{Filters: f})
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true, Filters: f})
```

Comment:
Same question here re: listing all containers vs. just the ones that match the filter.

Reply:
See previous answer. "All" means "including stopped"